### PR TITLE
Update rubygem-foreman_ansible to 6.2.0

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by xprazak2** [comment]
[test rpm]

File: packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 1]
```diff
@@ -42,15 +42,8 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
...
-BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) >= 3.0.0
-BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) < 4.0.0
-BuildRequires: %{?scl_prefix}npm(react-redux) >= 5.0.7
-BuildRequires: %{?scl_prefix}npm(react-redux) < 6.0.0
-BuildRequires: %{?scl_prefix}npm(redux) >= 3.7.2
-BuildRequires: %{?scl_prefix}npm(redux) < 4.0.0
-BuildRequires: %{?scl_prefix}npm(redux-thunk) >= 2.3.0
-BuildRequires: %{?scl_prefix}npm(redux-thunk) < 3.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 6.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 7.0.0
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec:46]: please drop the upper bound, we have builder 8 right now

**Comment by evgeni** [comment]
repoclosure unrelated

**Review by evgeni — APPROVED** [review]

**Pull request merged**